matchMinima.py

```diff
@@ -19,6 +19,8 @@
 
 
 
+#M Do what OE does and define your own molecule objects, they could even 
+#M extend the OE chem definitions to save you work. 
```

Comment:
OE seems to have a molecule object. If you wanted to you could define your own personal molecule object that inherits from the OE molecule. That would mean your new object would have all the methods and attributes an OE molecule object has, plus whatever you wanted to add. Declare it like `class myMolecule(OEMolecule):`